doc/development/index.md

```diff
@@ -0,0 +1,4 @@
+# Developer Guide
+   
+- [coding-conventions](coding-conventions.rst)
+- [api](api.rst)
```

Comment:
These two links use the filename instead of the title?

Reply:
I'll change that.